meson.build

```diff
@@ -40,12 +40,14 @@ project('picolibc', 'c',
 	  'b_staticpic=false',
 	],
 	license : 'BSD',
-	meson_version : '>= 0.50',
+	meson_version : '>= 0.53.2',
```

Comment:
I want this number to be as small as possible -- meson will complain if you're using features from later versions.

Reply:
The commit _"newlib/libc/meson.build: Use is_dir() in Filesystem module"_ is using a new feature [FS module, which comes with Meosn 0.53](https://mesonbuild.com/Release-notes-for-0-53-0.html#a-new-module-for-filesystem-operations).

Do you think it's worth bumping the supported version?